docs/src/main/asciidoc/spring-cloud-stream.adoc

```diff
@@ -712,7 +714,19 @@ For example, the above composition could be defined as such (if both functions p
 --spring.cloud.stream.function.definition=reactiveUpperCase|wrapInQuotes
 ----
 
+===== Batch Consumers
+
+When using a `MessageChannelBinder` that supports batch listeners, and the feature is enabled for the consumer binding, you can set `spring.cloud.stream.function.definition` to `true` to enable the entire batch of messages to be passed to the function in a `List`.
```

Comment:
Hmm, `spring.cloud.stream.function.definition`? Did you mean `spring.cloud.stream.bindings.input.consumer.batch-mode=true`?